### PR TITLE
Added doc comments modules

### PR DESCRIPTION
**Pull request by PeterYordanov**
Fixed spelling mistakes, added documentation comments for the modules, resulting in this:

image

### CODE REVIEW

File: src/cond.rs [hunk 1]
```diff
@@ -11,7 +14,7 @@ pub struct Cond {
 
 impl Cond {
     pub fn new() -> Result<Cond> {
-        let (rfd, wfd) = pipe2(OFlag::O_CLOEXEC)?;
+        let (rfd, wfd) = pipe2(OFlag::O_CLOEXEC)?; //Sets as close-on-execution
```
Comment (utam0k) [inline src/cond.rs:17]: Can you put a space after `//` following other comments?

Comment (tsturzl) [reply src/cond.rs:17]: Also putting the comment on a line above would be more inline with current formatting.

Comment (PeterYordanov) [reply src/cond.rs]: Updated, also added a one-sentence description for src/command module

File: src/utils.rs [hunk 2]
```diff
@@ -24,7 +26,7 @@ impl PathBufExt for PathBuf {
     fn join_absolute_path(&self, p: &Path) -> Result<PathBuf> {
         if !p.is_absolute() && !p.as_os_str().is_empty() {
             bail!(
-                "connnot join {:?} because it is not the absolute path.",
+                "connot join {:?} because it is not the absolute path.",
```
Comment (Furisto) [inline src/utils.rs:29]: c**a**nnot

**Review by utam0k — APPROVED** [review]
If you fix the parts that @Furisto points out, it looks good.

**Review by utam0k — APPROVED** [review]

**Pull request merged**